### DISCUSSION TITLE
Error on Merge Button

### DISCUSSION THREAD

**Discussion by TheTripleV** [community/community#37361] [Actions]
Merging a PR that changes an actions workflow fails with insufficient oauth scope.

![image](https://user-images.githubusercontent.com/11065157/198063243-0b1780bd-7b23-434b-92bd-ad4fb1671aad.png)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.85.0 (81053834)
Device: iPhone 12 Pro (iOS 16.0)
</details>

**Comment by ossanna16** [comment]
Hi there @TheTripleV and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by dragonflies4utoo** [comment]
Blessings to you, Lee Ann


---------- Forwarded message ---------
From: Lee Ann Coleman
***@***.***
>
Date: Wed, Nov 2, 2022 at 12:50 PM
Subject: Hello
To: ***@***.*** ***@***.***>


Ghosting


Get Outlook for iOS <https://aka.ms/o0ukef>

**Comment by dragonflies4utoo** [comment]
Help me out here

-- 
Blessings to you, Lee Ann

**Comment by dragonflies4utoo** [comment]
On Tue, Jan 10, 2023 at 2:36 AM Lee Ann Coleman ***@***.***>
wrote:

> Help me out here
>
> --
> Blessings to you, Lee Ann
>
> --
Blessings to you, Lee Ann

**Comment by dragonflies4utoo** [comment]
Lee Ann

Get Outlook for iOS<https://aka.ms/o0ukef>

**Comment by jsoref** [comment]
@TheTripleV you should change this to the [`category:Mobile`](https://github.com/community/community/discussions/categories/mobile). It _might_ result in the right team noticing.

Personally, after doing that, I'd open a ticket using https://support.github.com/
